modules/preparing-an-inital-cluster-deployment-for-mce-connected.adoc

```diff
@@ -0,0 +1,297 @@
+// Module included in the following assemblies:
+//
+// * installing_with_agent_based_installer/preparing-an-agent-based-installed-cluster-for-mce.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-an-inital-cluster-deployment-for-mce-connected_{context}"]
+
+= Preparing an agent-based cluster deployment for multicluster engine for Kubernetes (MCE) Operator while connected
+
+Create the required manifests for the multicluster engine for Kubernetes (MCE) Operator, Local Storage Operator (LSO), and to deploy an agent-based {product-title} cluster as a hub cluster.
+
+.Procedure
+
+. Create a sub-folder named  `openshift` in the `<assets_directory>` folder. This sub-folder is used to store the extra manifests that will be applied during the installation to further customize the deployed cluster.
```

Comment:
Folder is fine but usually "subdirectory" is used so suggested updating folder to directory to match
```suggestion
. Create a subdirectory named  `openshift` in the `<assets_directory>` directory. This subdirectory is used to store the extra manifests that will be applied during the installation to further customize the deployed cluster.
```